Automattic/wp-calypso
# Translate Odie correctly

This allows Odie to be translated in both Calypso and wp-admin contexts.

Currently, Odie is only translated in Calypso. This fixes that by adding dynamic context as explained here: pbAok1-1Ao-p2

### Testing
1. checkout this branch.
2. cd into `apps/editing-toolkit`.
3. run `yarn dev --sync`.
4. Change your user language to something non-English.
5. go to /wp-admin/post-new.php
6. open the Help Center.
7. Reach Odie.
8. Make sure the UI is translated.
